src/Akka.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27004.2010
+# Visual Studio Version 16
```

Comment:
Should probably bump the version of Visual Studio tooling we're using in a separate PR, but I'm open to doing that.

Reply:
I will revert that line. I committed it accidentally.

Reply:
For the record. This line does nothing. Its only a hint/marker for visual studio. If you commit this, and you open the sln in vs2015 it will change this line back automatically. I opened multiple bug reports with the VS team about this annoyance, and for some reason, its reintroduced again and again after a few VS updates.

Reply:
good to know, tnx